visualization/app/codeCharta/util/fileExtensionCalculator.ts

```diff
@@ -1,5 +1,5 @@
 import { CodeMapNode } from "../codeCharta.model"
-import { hierarchy } from "d3"
+import { hierarchy } from "d3-hierarchy"
```

Comment:
We don't have `d3-hierarchy` installed as a dependency. Should we remove `d3` and add `d3-hierarchy` instead to lower our bundle size?